src/e2e/resources/simple-temple-expected/simple-temple-test-user/dao/dao.go

```diff
@@ -167,3 +173,21 @@ func (dao *DAO) UpdateSimpleTempleTestUser(input UpdateSimpleTempleTestUserInput
 
 	return &simpleTempleTestUser, nil
 }
+
+// IdentifySimpleTempleTestUser returns the simpleTempleTestUser in the datastore for a given ID
+func (dao *DAO) IdentifySimpleTempleTestUser(input IdentifySimpleTempleTestUserInput) (*SimpleTempleTestUser, error) {
+	row := executeQueryWithRowResponse(dao.DB, "SELECT id, simple_temple_test_user, email, first_name, last_name, created_at, number_of_dogs, yeets, current_bank_balance, birth_date, breakfast_time FROM simple_temple_test_user WHERE id = $1;", input.ID)
+
+	var simpleTempleTestUser SimpleTempleTestUser
+	err := row.Scan(&simpleTempleTestUser.ID, &simpleTempleTestUser.SimpleTempleTestUser, &simpleTempleTestUser.Email, &simpleTempleTestUser.FirstName, &simpleTempleTestUser.LastName, &simpleTempleTestUser.CreatedAt, &simpleTempleTestUser.NumberOfDogs, &simpleTempleTestUser.Yeets, &simpleTempleTestUser.CurrentBankBalance, &simpleTempleTestUser.BirthDate, &simpleTempleTestUser.BreakfastTime)
+	if err != nil {
+		switch err {
+		case sql.ErrNoRows:
+			return nil, ErrSimpleTempleTestUserNotFound(input.ID.String())
+		default:
+			return nil, err
+		}
+	}
+
+	return &simpleTempleTestUser, nil
+}
```

Comment:
I get that it makes it easier to generate, but if you were writing this yourself, wouldn't you reuse the `read` DAO call?

Reply:
Yeah, I did want to do that but it didn't fit in with current generation 💔